include/zephyr/bluetooth/services/ias.h

```diff
@@ -17,10 +17,19 @@
  * as a part of ongoing development.
  */
 
+#include <zephyr/bluetooth/conn.h>
+
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/** @brief Immediate Alert Service alert levels */
+enum bt_ias_alert_lvl {
+	BT_IAS_ALERT_LVL_NO_ALERT,
+	BT_IAS_ALERT_LVL_MILD_ALERT,
+	BT_IAS_ALERT_LVL_HIGH_ALERT,
```

Comment:
We should consider adding a small description for these. I know that what the levels mean is highly contextual, but I assume that the specification has some description or recommendation on when to use one over the other?

Reply:
I've added a simple description based on specification - but don't know whether it's enoughor not.

Reply:
Please put the descriptions of the individual values as comments for each value, like what is done here: https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/bluetooth/bluetooth.h#L393-L583

That way it is properly picked up by doxygen